daemon/protocol/README.md

```diff
@@ -1,16 +1,34 @@
-The protocol betweeen Scoot Local Server and Scoot Local Clients.
+# Scoot Daemon Protocol
 
+The protocol betweeen Scoot Local Server and Scoot Local Clients.
 This includes the protocol buffers, and any other utilities that should be shared by client and server.
 
-// TODO(dbentley): integrate this into our go build
-// For now, to compile into daemon.pb.go, you need to:
-// install a protoc with libproto version >= 3.0.0 from https://github.com/google/protobuf
-// install protoc-gen-go from https://github.com/golang/protobuf (you need this to be on your $PATH, not your GOPATH)
-// in this directory (github.com/scootdev/scoot/daemon/protocol), run:
-// protoc -I . daemon.proto --go_out=plugins=grpc:.
-//
-// To compile into daemon_pb2.py:
-//  pip install grpcio-tools # Install latest protobuf and grpc+tools packages.
-//  brew uninstall protobuf # Be sure to remove any conflicting/older versions of protobuf.
-//  python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. daemon.proto
-//
+__This still needs to be integrated into the build__
+
+#### Compiling for Go (daemon.pb.go)
+
+* Install a protoc with libproto version >= 3.0.0 from https://github.com/google/protobuf
+* install protoc-gen-go from https://github.com/golang/protobuf (you need this to be on your $PATH, not your GOPATH)
+* In this directory (github.com/scootdev/scoot/daemon/protocol), run:
+```sh
+protoc -I . daemon.proto --go_out=plugins=grpc:.
+```
+
+#### Compiling for Python (daemon_pb2.py)
+
+* Install latest protobuf and grpc+tools packages:
+```sh
+pip install grpcio-tools
+```
+
+_Editor's note_: we should think about using a _virtualenv_ for a fully portable process, and to
```

Comment:
I think this may more overhead than it's worth. We currently require grpc 1.0.1 but we could also say it's a trustworthy package and accept anything greater than that. That wouldn't be too onerous...